charts/pod-director/templates/configmap.yaml

```diff
@@ -1,3 +1,22 @@
+{{- $labelKeyRegex := "^([a-z0-9]([-a-z0-9]*[a-z0-9])?\\.)*([a-z0-9]([-a-z0-9]*[a-z0-9])?\\/)?[a-z0-9]([-a-z0-9]*[a-z0-9])?$" }}
+{{- $groupLabel := .Values.config.groupLabel }}
+{{- if $groupLabel }}
+  {{- if gt (len $groupLabel) 63 }}
+    {{- fail (printf "The group label %s is longer than the maximum length 63 for labels" $groupLabel) }}
+  {{- end }}
+  {{- if not (mustRegexMatch $labelKeyRegex $groupLabel) }}
+    {{- fail (printf "The group label %s is not a valid label" $groupLabel) }}
+  {{- end }}
+{{- end }}
+{{- $labelValueRegex := "^[a-z0-9]([-a-z0-9]*[a-z0-9])?$" }}
+{{- range $groupName := keys .Values.config.groups }}
+  {{- if gt (len $groupName) 63 }}
+    {{- fail (printf "The group name %s is longer than the maximum length 63 for label values" $groupName) }}
+  {{- end }}
+  {{- if not (mustRegexMatch $labelValueRegex $groupName) }}
+    {{- fail (printf "The group name %s is not a valid label value and thus cannot be a group name" $groupName) }}
```

Comment:
Add quotes for `%s` to be more visible in the error message the value of $groupName